sysl2/sysl/seqs_test.go

```diff
@@ -398,3 +406,108 @@ func TestDoGenerateSequenceDiagrams(t *testing.T) {
 		})
 	}
 }
+
+func TestDoConstructSequenceDiagramWithGroupingCommandline(t *testing.T) {
+	// Given
+	args := &sdArgs{
+		rootModel: "./tests/",
+		modules:   "call.sysl",
+		output:    "tests/call.png",
+		endpoints: []string{"MobileApp <- Login"},
+		groupbox:  "owner",
+	}
+	var boxPresent bool
+	var err error
+
+	// When
+	boxServer := `box "server" #LightBlue
+	participant _\d
+	participant _\d
+end box`
+	boxClient := `box "client" #LightBlue
+	participant _\d
+	participant _\d
+end box`
+	result := DoConstructSequenceDiagrams(args.rootModel, args.endpointFormat, args.appFormat,
+		args.title, args.output, args.modules, args.endpoints, args.apps, args.blackboxes, args.groupbox)
+
+	// Then
+	boxPresent, err = regexp.MatchString(boxServer, result["tests/call.png"])
+	if err != nil {
```

Comment:
replace 
```go
if err != nil {
...
}
```
with

```go
  assert. NotNil(t, err, "....")
```

Reply:
I think you mean assert.Nil(t, err, "....")